### PR TITLE
Feat: Replace `aria2` with `axel` for faster speeds

### PR DESCRIPTION
**Pull request by NarutoXY**
Go crazy.

### CODE REVIEW

File: misc/scripts/install-local.sh [hunk 1]
```diff
@@ -92,9 +92,13 @@ function loggingMeta() {
 
 
 function aria2 {
+Parallel=$nproc + 5
```
Comment (D-Brox) [inline misc/scripts/install-local.sh:95]: I think it's best to use `bc` for safe string operation

Comment (D-Brox) [reply misc/scripts/install-local.sh:95]: Codefactor has a better option

Comment (NarutoXY) [reply misc/scripts/install-local.sh:95]: May you edit this?? I'm not familiar with bc

Comment (D-Brox) [reply misc/scripts/install-local.sh]: Codefactor had a better option, actually

Comment (NarutoXY) [reply misc/scripts/install-local.sh]: Ok, did changes

**Comment by wizard-28** [comment]
Hey what about `-j`?
img

**Comment by D-Brox** [comment]
@wizard-28  `-j` is for concurrent downloads, so multiple files. We can use it on `install.sh` and `update.sh` for downloading all scripts simultaneously, tho

**Comment by NarutoXY** [comment]
> Hey what about `-j`?
> img

Have mercy on CPUs and ```-j``` will be used in install.sh and update.sh as @D-Brox pointed out its for multi file download

**Pull request closed without merging**